MeshGeoToolsLib/MeshNodesOnPoint.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * @copyright
```

Comment:
It is possible to figure out this information via `git blame ...`. For debugging it is more important how wrote or changed the particular lines.